integration-tests/baas-test-server/index.ts

```diff
@@ -17,40 +17,24 @@
 ////////////////////////////////////////////////////////////////////////////
 
 import cp, { execSync } from "node:child_process";
-import path from "path";
-import fs from "fs";
 import chalk from "chalk";
 import dotenv from "dotenv";
 import assert from "node:assert";
-import { Socket } from "node:net";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
 
-const MONGO_CONTAINER_NAME = "mongo";
-const STITCH_SUPPORT_URL =
-  "https://stitch-artifacts.s3.amazonaws.com/stitch-support/macos-arm64/stitch-support-6.1.0-alpha-527-g796351f.tgz";
-const BAAS_REPO = "git@github.com:10gen/baas.git";
-const BAAS_UI_REPO = "git@github.com:10gen/baas-ui.git";
-const MONGODB_PORT = 26000;
-const MONGODB_URL = `mongodb://localhost:${MONGODB_PORT}`;
-const ASSISTED_AGG_URL =
-  "https://stitch-artifacts.s3.amazonaws.com/stitch-mongo-libs/stitch_mongo_libs_osx_patch_df9a68d900d7faddcee16bf0f532437da815a1c3_63289ac95623436dd66f7056_22_09_19_16_37_32/assisted_agg";
+const IMAGES_ENDPOINT = "https://us-east-1.aws.data.mongodb-api.com/app/baas-container-service-autzb/endpoint/images";
+const ECR_HOSTNAME = "969505754201.dkr.ecr.us-east-1.amazonaws.com";
+
+const BAAS_CONTAINER_NAME = "baas-test-server";
+const BAAS_PORT = 9090;
 
 dotenv.config();
 
-const currentDirPath = path.dirname(new URL(import.meta.url).pathname);
-const baasPath = path.resolve(currentDirPath, "baas");
-const staticPath = path.resolve(baasPath, "static");
-const staticAppPath = path.resolve(staticPath, "app");
-const dylibPath = path.resolve(baasPath, "etc/dylib");
-const transpilerPath = path.resolve(baasPath, "etc/transpiler");
-const transpilerBinPath = path.resolve(transpilerPath, "bin");
-const baasTmpPath = path.resolve(baasPath, "tmp");
-const dylibFilename = "libstitch_support.dylib";
-const dylibUsrLocalLibPath = "/usr/local/lib/" + dylibFilename;
-const assistedAggPath = path.resolve(baasPath, "assisted_agg");
+const { AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY } = process.env;
```

Comment:
Does it mean that you don't need to store them in `.env`?

Reply:
Yep - or provide them to the process on the command-line, but the `dotenv.config()` above helps load them from a file.